lmnet/tests/lmnet_tests/datasets_tests/test_div2k.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import os
+from glob import glob
+
+import numpy as np
+import pytest
+
+from lmnet import environment
+from lmnet.datasets.dataset_iterator import DatasetIterator
+from lmnet.datasets.div2k import Div2k
+
+
+def test_train_files(set_test_environment):
+    dataset = Div2k("train")
```

Comment:
I think we don't need to bind everything into local variables.

`assert sorted(Div2k("train")) == expected` is short and easy to understand.

Same for other tests.